src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -106,13 +108,18 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   override def adjustInputPaths(call: Call, inputs: CallInputs): CallInputs = inputs map { case (k, v) => (k, gcsPathToLocal(v)) }
   override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs map { case (k,v) => (k, gcsPathToLocal(v)) }
 
+  private def makeLogger(descriptor: WorkflowDescriptor) = {
```

Comment:
I'm waffling here as this feels like it belongs in the companion object but obviously the getClass portion requires some class-local angle